### ISSUE TITLE
Unable to run "devspace use space" command in CircleCI build

### ISSUE THREAD

**Issue by slevine111** [devspace-cloud/devspace#763]
**What happened?**  
When I try running the "devspace use space [name of space]" in a CircleCI build, I get an error.  See below for my config.yml for CircleCI and the messages for successful devspace related commands (installing it and devspace login) and the error message for devspace use space

<img width="1169" alt="SuccessfulDevspaceSteps" src="https://user-images.githubusercontent.com/24880825/67624145-16083780-f7fb-11e9-918c-96d420e302ea.png">

<img width="1166" alt="DevspaceusespaceError" src="https://user-images.githubusercontent.com/24880825/67624150-1acceb80-f7fb-11e9-85eb-34b802db050b.png">

```
version: 2
jobs:
  build_and_deploy:
    docker:
      # specify the version you desire here
      - image: circleci/buildpack-deps:jessie

    steps:
      - checkout

      - run:
          name: put environment variables in secrets
          command: |
            POSTGRES_USER_ENCODED=$(echo $POSTGRES_USER | base64)
            POSTGRES_PASSWORD_ENCODED=$(echo $POSTGRES_PASSWORD | base64)
            REDIS_PASSWORD_ENCODED=$(echo $REDIS_PASSWORD | base64)
            JWT_SECRET_ENCODED=$(echo $JWT_SECRET | base64)
            jwt_secret_lines=$(echo $JWT_SECRET_ENCODED | tr " " "\n")
            sed -i "s/ENV-VAR-POSTGRES-USER/${POSTGRES_USER_ENCODED}/" ./k8s/secret.yaml
            sed -i "s/ENV-VAR-POSTGRES-PASSWORD/${POSTGRES_PASSWORD_ENCODED}/" ./k8s/secret.yaml
            sed -i "s/ENV-VAR-REDIS-PASSWORD/${REDIS_PASSWORD_ENCODED}/" ./k8s/secret.yaml
            for line in $jwt_secret_lines
            do
                sed -i "s/ENV-VAR-JWT-SECRET/${line}ENV-VAR-JWT-SECRET/" ./k8s/secret.yaml
            done
            sed -i "s/ENV-VAR-JWT-SECRET//" ./k8s/secret.yaml

      - run:
          name: install devspace
          command: |
            curl -s -L "https://github.com/devspace-cloud/devspace/releases/latest" |\
            sed -nE 's!.*"([^"]*devspace-linux-amd64)".*!https://github.com\1!p' | xargs -n 1 curl -L -o devspace && chmod +x devspace;
            sudo mv devspace /usr/local/bin;

      - run:
          name: devspace login
          command: devspace login --key ${DEVSPACE_CLOUD_ACCESS_KEY}

      - run:
          name: set up space to use
          command: devspace use space worldofchatting

      - run:
          name: deploy
          command: devspace deploy

  test:
    docker:
      # specify the version you desire here
      - image: circleci/buildpack-deps:jessie

    steps:
      - checkout
      - setup_remote_docker

      - run:
          name: build test image
          command: docker-compose build

      - run:
          name: test
          command: docker-compose run webserver npm run test

      - run:
          name: containers
          command: |
            docker container ls
            docker container ls -a

      - run:
          name: copy coverage file
          command: docker cp worldofchatting-webserver-test:/user/src/app/coverage-circleci ./coverage-circleci

      - run:
          name: check
          command: |
            pwd
            ls

      - store_artifacts:
          path: ./coverage-circleci

workflows:
  version: 2
  test_build_and_deploy:
    jobs:
      - test
      - build_and_deploy:
          requires:
            - test
          filters:
            branches:
              ignore: master
```

**What did you expect to happen instead?**  
"devspace use space worldofchatting" to have run successfully, as it does when running it on my computer

**How can we reproduce the bug?** (as minimally and precisely as possible)  
1. create a CircleCI project
2. create a devspace space
3. create a config.yml with installing devspace, logging in, and then running "devspace use space [name of space]"

**CircleCI Docker container**  
- base image using in config.yml: circleci/buildpack-deps:jessie ([https://github.com/CircleCI-Public/circleci-dockerfiles/tree/master/buildpack-deps/images/jessie](https://github.com/CircleCI-Public/circleci-dockerfiles/tree/master/buildpack-deps/images/jessie))

/kind bug

**Comment by FabianKramm** [comment]
@slevine111 thanks for opening this issue! I will add a fix to this issue to the next release

**Comment by slevine111** [comment]
@FabianKramm is the issue solved or did you reopen this b/c still working on it? thanks for the update

**Comment by slevine111** [comment]
@FabianKramm ok sounds good. Thanks for the update

**Comment by FabianKramm** [comment]
@slevine111 should be fixed in v4.2.0
